newrelic-infra.dockerfile

```
@@ -0,0 +1,2 @@
+FROM newrelic/infrastructure:latest
+ADD backend/newrelic-infra/newrelic-infra.yml /etc/newrelic-infra.yml
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)